tooling/nargo_cli/src/cli/mod.rs

```diff
@@ -75,6 +76,8 @@ enum NargoCommand {
     Lsp(lsp_cmd::LspCommand),
     #[command(hide = true)]
     Dap(dap_cmd::DapCommand),
+    #[command(hide = true)]
+    Fv(fv_cmd::FvCommand),
```

Comment:
The first `Fv` on this line is the fv that becomes the CLI command. Please rename to FormalVerify; this will result in the command `nargo formal-verify`. We can further add an alias `fv` to make it easier to call. (Search for alias in prove_cmd.rs.)

Reply:
Done